internal/pods/schedule.go

```diff
@@ -124,7 +130,7 @@ func (np *Scheduled) schedule() error {
 			Containers: []v1.Container{
 				{
 					Name:    np.Config.Name,
-					Image:   "quay.io/submariner/nettest:devel",
+					Image:   fmt.Sprintf("%s/nettest:devel", imageRepo),
```

Comment:
I think this should also use the right version tag, for a variety of reasons; at least (a) we can’t expect `nettest:devel` to always work for us (on a release branch); and (b) we can’t expect users to pull a `:devel` image when they prepare a set of images for disconnected provisioning, instead of whatever version they’re setting up. See `images.GetImagePath()`, I think that would be useful here.

Reply:
ok - makes sense. I changed it to extract the image registry info from the `Submariner` CRD into an `image.RepositoryInfo` struct and use `images.GetImagePath(...)`.